src/app/marketplace/marketplace-product-details.component.html

```diff
@@ -33,52 +45,101 @@ <h1>{{ product.title }}</h1>
         [availableActions]="availableActions"></app-marketplace-action-buttons>
     </section>
 
+    <section class="product-details__summary">
+      <a
+        *ngIf="sellerRating.toNumber() > 0"
+        (click)="scrollIntoView('seller-rating')"
+         class="product-details__summary-item product-details__summary-item_link">
+        <span class="product-details__summary-number">
+          {{ sellerRating.toNumber() | percent }}
+        </span>
+        <span class="product-details__summary-label">
+         Positive feedback
+        </span>
+      </a>
+
+      <div class="product-details__summary-item" *ngIf="sellerRating.toNumber() == 0">
+        <span class="product-details__summary-label">
+         No feedback yet
+        </span>
+      </div>
+
+      <a *ngIf="product.sampleFile?.length > 0"
+        (click)="scrollIntoView('sample-files')"
+        class="product-details__summary-item product-details__summary-item_link">
+        <span class="product-details__summary-number">
+          {{ product.sampleFile.length }}
+        </span>
+        <span class="product-details__summary-label" [ngPlural]="product.sampleFile.length">
+         Sample
+          <ng-template ngPluralCase="=0">files</ng-template>
+          <ng-template ngPluralCase="=1">file</ng-template>
+          <ng-template ngPluralCase="other">files</ng-template>
+        </span>
+      </a>
+
+      <div class="product-details__summary-item" *ngIf="!product.sampleFile">
+        <span class="product-details__summary-label">
+         No sample files provided
+        </span>
+      </div>
+
+      <div (click)="scrollIntoView('rules')"
+           class="product-details__summary-item product-details__summary-item_link">
+        <span class="product-details__summary-number">
+          {{ product.daysToDeliver }}
+        </span>
+        <span class="product-details__summary-label" [ngPlural]="product.daysToDeliver">
+          <ng-template ngPluralCase="=0">Days</ng-template>
+          <ng-template ngPluralCase="=1">Day</ng-template>
+          <ng-template ngPluralCase="other">Days</ng-template>
+          to deliver
+        </span>
+      </div>
+      <div class="product-details__summary-item">
+        <span class="product-details__summary-number">
+          {{ product.orders.length }}
+        </span>
+        <span class="product-details__summary-label" [ngPlural]="product.orders.length">
+          <ng-template ngPluralCase="=0">Items</ng-template>
```

Comment:
In my opinion it would be better if you change message from '1 Items bought' to e.g. '1 time purchased' because there is only one product on this page.